src/common/utils.ts

```diff
@@ -102,3 +102,13 @@ export function camelCaseToSnakeCase(
 export function debeziumTimeToDate(time: number): Date {
   return new Date(Math.floor(time / 1000));
 }
+
+export const ensureMsFormat = (value: number): number => {
```

Comment:
To support the tests that are not able to replicate the same value that debezium provides.